src/kartikeyDJune20/PhoneNumValidations.java

```diff
@@ -0,0 +1,35 @@
+package kartikeyDJune20;
+
+public class PhoneNumValidations {		
+		void phoneNumValidation(String number) {
+			int numLength=number.length();
+			boolean isNumValid=false;
+			if(numLength>=10) {
+				for(int index=0;index<number.length();index++) {			
+					if(numLength==11 && number.charAt(0)=='0') {
```

Comment:
here making flag value as true, not proper logic, like If in between once the flag value set as false but again loop iterating and for next value flag comes true but if once the flag value is false then you need to break n say that number is invalid.